packages/circular-view/src/StructuralVariantChordTrack/models/renderReaction.js

```diff
@@ -12,14 +10,11 @@ export default ({ jbrequire }) => {
     const { rendererType, renderProps } = track
     const { rpcManager } = getSession(view)
 
-    // TODO: do multiple assemblies
-    const [assemblyName] = getTrackAssemblyNames(track)
-    const data = {
```

Comment:
note: I think the procedure of manually passing an assemblyName is outdated as region will contain assemblyName

Reply:
hence, this diff